### ISSUE TITLE
Reg ex issue found in avstream for windows path.sep charaters

### ISSUE THREAD

**Issue by martynadams007** [iizukanao/node-rtsp-rtmp-server#29]
Hi 

When playing back a recorded mp4 file using the new features you've recently added (thanks for that!), I get an error relating to the regEx expression around line 350 (avstream.coffee)

Error: 
**SyntaxError: Invalid regular expression: /(^|\)..(\|$)/: Unterminated group**

I believe is only seen when running node under windows and the fact that the path separator is a backslash and needs to be escaped within a regEx expression.

I added a quick fix local fix

```javascript
 # Check that filename is legitimate
          pathSep = path.sep
          pathSep += "\\" if (pathSep == "\\")                
                
          if (filename is '.') or
          new RegExp("(^|#{pathSep})..(#{pathSep}|$)").test filename
            logger.warn "rejected request to stream: #{streamId}"
            break
````

I've only tested this on windows 8.

Thanks

**Comment by iizukanao** [comment]
@martynadams007 Thanks for reporting the bug! Fixed at 79018dc85852e4506f9fcbdfcbfd39aabacb1cdc.

**Comment by martynadams007** [comment]
Brilliant, thanks.
